packages/devtools_app/lib/src/screens/deep_link_validation/select_project_view.dart

```diff
@@ -88,7 +88,7 @@ class _SelectProjectViewState extends State<SelectProjectView>
           return const DevToolsDialog(
             title: Text('You selected a non Flutter project'),
             content: Text(
-              'Seems you selected a non Flutter project. If it is not intended, please reselect a Flutter project.',
+              'Seems you selected a non-Flutter project. If it is not intended, please reselect a Flutter project.',
```

Comment:
wording nit: "It looks like you have selected a non-Flutter project. Please select a Flutter project instead."